auxiliary/stata-guide.tex

```diff
@@ -106,32 +72,82 @@ \subsection{Writing and Understanding Stata code}
 (or much less) accessible to someone who is reading it for the first time
 and needs to understand it quickly and correctly.
 
-Whether you are new to Stata or have used it for decades,
+Whether you are new to the programming language you are suing,
+(Stata, R, Python etc.)
+or have used it for decades,
 you will always run into commands that
 you have not seen before or whose function you do not remember.
-(Whether you are new or not, you should frequently revisit the most common commands --
-often you will learn they can do something you never realized.\sidenote{
-  \url{https://www.stata.com/manuals13/u27.pdf}})
 Every time that happens,
 you should always look up the help file for that command.
 We often encounter the conception that help files are only for beginners.
 We could not disagree with that conception more,
-as the only way to get better at Stata is to constantly read help files.
-So if there is a command that you do not understand in any of our code examples,
-for example \texttt{isid}, then write \texttt{help isid},
-and the help file for the command \texttt{isid} will open.
+as the only way to get better at the programming language you use
+is to constantly read help files.
+(Whether you are new or not, you should frequently revisit the most common commands --
+often you will learn they can do something you never realized.)
 We cannot emphasize enough how important it is
 that you get into the habit of reading help files.
 Most of us have a help file window open at all times.
 
```

Comment:
This section ends a bit abruptly, needs some sort of conclusion. I will take a stab at it.